src/test/java/org/spongepowered/common/registry/CatalogTypeMethodTest.java

```diff
@@ -103,22 +84,13 @@
     @Test
     public void testCatalogMethodImpl() throws Throwable {
         try {
-            try {
-                testResult(checkNotNull(this.method.invoke(this.catalogType), "return value"));
-            } catch (InvocationTargetException e) {
-                // Unwrap exception to avoid useless stacktrace entries
-                if (e.getCause() != null) {
-                    throw e.getCause();
-                }
-                throw e;
-            }
-        } catch (Throwable t) {
-            if (ignoredFailures.contains(this.implementationClass + "#" + this.method.getName() + "()")) {
-                LOG.warn("Catalog Type: {} Id : {} has broken Method: {} ({}): {}", this.name, this.catalogId, this.methodName,
-                        this.implementationClass, t);
-                return;
+            testResult(checkNotNull(this.method.invoke(this.catalogType), "return value"));
```

Comment:
How is this changing how it used to work previously?

Reply:
Previously, everything inside `ignoredFailures`would be ignored (it would just log "x has broken method: etc.") but it was a workaround (see the TODO above). I've just removed the `if` and unwrapped the exception.